mne/simulation/raw.py

```diff
@@ -74,11 +91,11 @@ def simulate_raw(raw, stc, trans, src, bem, cov='simple',
     bem : str | dict
         BEM solution  corresponding to the stc. If string, should be a BEM
         solution filename (e.g., "sample-5120-5120-5120-bem-sol.fif").
-    cov : instance of Covariance | str | None
-        The sensor covariance matrix used to generate noise. If None,
-        no noise will be added. If 'simple', a basic (diagonal) ad-hoc
-        noise covariance will be used. If a string, then the covariance
-        will be loaded.
+    cov : instance of Covariance | str | dict | 'simple' | None
```

Comment:
`'simple`' is a `str`.

Reply:
Yes, I'll specify it in the explanation.